music/views.py

```diff
@@ -47,3 +58,71 @@ def get(self, request, media_id):
                                      status=200, content_type='audio/mp3')
         resp['Cache-Control'] = 'no-cache'
         return resp
+
+class MainView(View):
+
+    def get(self, request):
+
+        range_list = [1,7,13]
+
+        if request.GET :
+            range_list = request.GET.getlist(
+                'collection_id'
+            )
```

Comment:
if request.GET : 구문은 꼭필요할까요 ? 
```suggestion
            range_list = request.GET.getlist('collection_id')
```

Reply:
저 부분의 if 문은 페이지네이션이 등장하기 전, 첫 GET 요청에 대한 처리를 위한 조건구문입니다.

저 것을 없애버리면,
1. 첫 GET 요청의 payload 가 빈 상태가 됩니다.
2. ' 1 ' 상태이기 때문에, 메인화면의 썸네일을 보내기 위한 기준 collection 이 사라집니다.

첫 요청에 대한 처리를 위한 더 효율적인 방법을 생각하지 못해, if 구문으로 처리했습니다... ㅠㅜ